src/shared/ui/survey/SingleSelectionPerRowItem/SingleSelectionPerRowItem.tsx

```diff
@@ -0,0 +1,191 @@
+import { FC, useState } from 'react';
+import { FlatList } from 'react-native';
+
+import { styled, Text } from '@tamagui/core';
+import { useTranslation } from 'react-i18next';
+
+import { colors } from '@app/shared/lib';
+
+import {
+  SingleSelectionPerRowConfigType,
+  SingleSelectionPerRowItemValue,
+} from './types';
+import {
+  Center,
+  XStack,
+  YStack,
+  Image,
+  Input,
+  Tooltip,
+  RadioGroup,
+  ListSeparator,
+} from '../..';
+
+const SingleSelectionPerRowAxisListItemContainer = styled(Center, {
```

Comment:
If a component is **not** supposed to be exportable then you can come up with any meaningful name that may be short.
For example "Container" or "ItemContainer"